service/src/main/kotlin/io/provenance/api/domain/usecase/provenance/tx/scope/ChangeScopeOwnership.kt

```diff
@@ -48,13 +48,13 @@ class ChangeScopeOwnership(
                         owner.role == PartyType.PARTY_TYPE_OWNER
                     }.forEach { existingOwner ->
                         scopeBuilder.removeOwners(scopeBuilder.ownersList.indexOf(existingOwner))
+                        scopeBuilder.addOwners(
+                            Party.newBuilder().also { ownerPartyBuilder ->
+                                ownerPartyBuilder.role = PartyType.PARTY_TYPE_OWNER
+                                ownerPartyBuilder.address = requestedNewValueOwner
+                            }.build()
+                        )
                     }
-                    scopeBuilder.addOwners(
-                        Party.newBuilder().also { ownerPartyBuilder ->
-                            ownerPartyBuilder.role = PartyType.PARTY_TYPE_OWNER
-                            ownerPartyBuilder.address = requestedNewValueOwner
-                        }.build()
-                    )
```

Comment:
> [only update owners list if using PARTY_TYPE_OWNER (metadata asset model)](https://github.com/provenance-io/p8e-cee-api/pull/65/commits/b9eacd530c4d82d5c7d452bfd83bc5bf862f3460)

Very confused. What does this change have to do with the MAM, and why is this snippet being moved into the loop when it doesn't depend on the context of `existingOwner`?